### DISCUSSION TITLE
Dax Studio - Export linked file to excel gives error

### DISCUSSION THREAD

**Discussion by NoahLull** [DaxStudio/DaxStudio#1096] [Q&A] [answered]
I am trying to export a linked DAX query from DAX studio, and it is giving me an error. We have premium capacity, and I am able to connect via the linked export file from PBI online and would like to be able to do the same from Dax Studio. It appears that I need to install a driver, but don't know where to start. Help is appreciated!

![Excel error for dax studio linked file](https://user-images.githubusercontent.com/110566226/231912437-5b97cb77-70ed-4565-b9e0-bb6097f256c4.jpg)

**Comment by marcosqlbi** [comment]
Can you connect Excel to a Power BI dataset (without DAX Studio)?

**Reply by NoahLull** [reply]
Yes, I can.

**Comment by dgosbell** [comment, marked as answer]
This is most likely related to issue #1083 where the generated .odc file does not work with AAD authentication. This will be fixed in the next release.

**Reply by NoahLull** [reply]
This is it! Thank you for the link.  When will the next release drop?  I truly love DAX studio! This is a game changer that I am only just scratching the surface on!

**Reply by dgosbell** [reply]
I should get the next update out tomorrow. I've just been doing some final checks and updating the change log today ready publish it.
